lib/modules/manager/bazel-module/readme.md

```diff
@@ -0,0 +1,8 @@
+<!-- prettier-ignore -->
+!!! warning
+    The `bazel-module` manager is a work-in-progress.
+    It is currently disabled.
```

Comment:
Do you _really_ mean that the `bazel-module` manager is disabled? That would mean this is a feature which never works...

I guess you mean this:

```suggestion
    By default, this manager is disabled.
    You can opt-in by INSERT INSTRUCTIONS HERE.
```

Reply:
I don't want anyone to use it. I will enable it with a follow-on PR. I want some additional features merged before the public uses it. I did not include all of the other features to keep this PR a reasonable size.